src/pages/Artist/Artist.js

```diff
@@ -1,15 +1,34 @@
 import React from 'react'
 import { useTranslation } from 'react-i18next'
-import { PageTitle } from '../../components'
 import { setPageTitle } from '../../utils/setPageTitle'
+import { useArtist } from '../../services/artists/query'
 
-export default function Artist() {
+export default function Artist(artistSlug) {
   const { t } = useTranslation()
+  const {
+    name,
+    slug,
+    description,
+    image_url,
+    fb_link,
+    sc_link,
+    vimeo_link,
+    yt_link,
+    ig_link,
+    created_at,
+  } = useArtist(artistSlug)
 
   React.useEffect(() => {
-    setPageTitle(t('artists'))
+    setPageTitle(name)
```

Comment:
I know that this thing dosnt properly work, because setPageTitle looks like executed before the api call. but we don't care I think this is not necessary to lose time with this for the moment, or maybe you have a quickly solution in your pocket